Jenkinsfile

```diff
@@ -1,52 +1,123 @@
-@Library('xmos_jenkins_shared_library@v0.14.2') _
+@Library('xmos_jenkins_shared_library@v0.15.1') _
 
 getApproval()
 
 pipeline {
-  agent {
-    label 'x86_64&&brew'
-  }
+  agent none
+  //Tools for AI verif stage. Tools for standard stage in view file
+  parameters {
+     string(
+       name: 'TOOLS_VERSION',
+       defaultValue: '15.0.2',
+       description: 'The tools version to build with (check /projects/tools/ReleasesTools/)'
+     )
+   }
   environment {
```

Comment:
I would have the environment blocks specified the other way around. TOOLS_PATH only makes sense on the NUC so define it there but REPO and VIEW apply to the whole pipeline so define it at the top.